docs/quickstart-examples.md

```diff
@@ -0,0 +1,105 @@
+# Quickstart Examples
+
+Now that you have installed the basic requirements, you can start pulling data. Try out the examples below, or navigate to the [api/client/samples/](https://github.com/gro-intelligence/api-client/blob/development/api/client/samples) folder for more options.
+
+## Get Started in 120 Seconds
+
+If you have a google account, or if you already have jupyter notebooks and python installed, and a Gro API token, you can start pulling Gro data via our API client within 120 seconds.

```

Comment:
1. Capitalize "Google"
2. "Jupyter Notebook" capitalized and singular

Reply:
Also link Jupyter Notebook to https://jupyter.readthedocs.io/en/latest/install.html ?

Reply:
Done.